src/CMakeLists.txt

```diff
@@ -134,3 +134,22 @@ install(
     ARCHIVE DESTINATION lib/static COMPONENT applications
     INCLUDES DESTINATION include COMPONENT applications
 )
+
+# Configure appimagetool.AppImage component
+install(
+    TARGETS appimagetool AppRun
+    RUNTIME DESTINATION bin COMPONENT appimagetool.AppImage EXCLUDE_FROM_ALL
+)
+
+find_library(LIBFFI_PATH NAMES ffi)
+find_program(ZSYNCCMAKE_PATH zsyncmake HINTS /usr/bin DOC "Build control file for zsync.")
+find_program(DESKTOP_FILE_VALIDATE_PATH desktop-file-validate HINTS /usr/bin)
+
+install(PROGRAMS ${ZSYNCCMAKE_PATH} DESTINATION bin COMPONENT appimagetool.AppImage EXCLUDE_FROM_ALL)
+install(PROGRAMS ${DESKTOP_FILE_VALIDATE_PATH} DESTINATION bin COMPONENT appimagetool.AppImage EXCLUDE_FROM_ALL)
+install(PROGRAMS ${mksquashfs_INSTALL_DIR}/mksquashfs DESTINATION lib/appimagekit/ COMPONENT appimagetool.AppImage EXCLUDE_FROM_ALL)
+
+install(FILES ${LIBFFI_PATH} DESTINATION lib COMPONENT appimagetool.AppImage EXCLUDE_FROM_ALL)
+install(FILES "${PROJECT_SOURCE_DIR}/resources/appimagetool.desktop" DESTINATION share/applications COMPONENT appimagetool.AppImage EXCLUDE_FROM_ALL)
+install(FILES "${PROJECT_SOURCE_DIR}/resources/appimagetool.svg" DESTINATION share/icons/hicolor/scalable/apps COMPONENT appimagetool.AppImage EXCLUDE_FROM_ALL)
+install(FILES "${PROJECT_SOURCE_DIR}/resources/usr/share/metainfo/appimagetool.appdata.xml" DESTINATION share/metainfo COMPONENT appimagetool.AppImage EXCLUDE_FROM_ALL)
```

Comment:
Missing newline... Please **finally** fix your IDE...

Reply:
Found it and fixed.

Reply:
Good.